packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1,31 +1,30 @@
 import * as React from 'react';
-
-import { CommandButton, IButtonStyles, IconButton } from '../../Button';
-import { Callout } from '../../Callout';
-import { Checkbox } from '../../Checkbox';
-import { KeytipData } from '../../KeytipData';
-import { Label } from '../../Label';
+import { Autofill, IAutofill } from '../Autofill/index';
```

Comment:
my VSO plugin reordered these imports and alphabetized them. I can revert this part if its unwanted.